tasks/deploy.py

```diff
@@ -121,7 +110,13 @@ def _get_app_key(cfg: Optional[Config]) -> str:
     return _get_key("APP KEY", cfg, lambda c: c.get_agent().appKey, "E2E_APP_KEY", 40)
 
 
-def _get_key(key_name: str, cfg: Optional[Config], get_key: Callable[[Config], Optional[str]], env_key_name: str, expected_size: int) -> str:
+def _get_key(
+    key_name: str,
+    cfg: Optional[Config],
+    get_key: Callable[[Config], Optional[str]],
+    env_key_name: str,
+    expected_size: int,
+) -> str:
```

Comment:
We should add a formater in order to not change the formatting of the code in each PR.